content/post/35-ipfs-companion-2.2.0.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2018-04-10
+url: 35-ipfs-companion-2-2-0
+title: IPFS Companion 2.2.0 brings window.ipfs to your Browser
+author: lidel
+---
+
+I am pleased to announce to everyone in our community that a new version of our
```

Comment:
You're using `us` and `we` in the rest of the post, so this should probably changed to be the same.